### ISSUE TITLE
Shield blocking

### ISSUE THREAD

**Issue by tulzke** [jrbudda/Vivecraft_119#13]
Is it technically possible to make blocking with a shield with the help of a hand, and not by pressing a button?

**Comment by lumithespoofy64** [comment]
No. But you could use the mod ImmersiveMC with vivecraft for that. ImmersiveMC makes VR immersive. Its goal is to remove the GUIs in minecraft and make it so its interactive.
